apps/trackers/jira/query.py

```diff
@@ -128,6 +128,17 @@ def generate_labels(self):
             )
         )
 
+        # If all affects are NEW and if this tracker is new, add label validation-requested.
+        if (
+            not self.tracker.external_system_id
+            or "validation-requested" in self.tracker.meta_attr.get("labels", [])
+        ):
+            affect_states = list(
+                self.tracker.affects.all().values_list("affectedness", flat=True)
+            )
+            if set(affect_states) == set([Affect.AffectAffectedness.NEW]):
+                self._query["fields"]["labels"] += ["validation-requested"]
+
```

Comment:
To avoid unnecessary nested `if` and `set(list())` constructions for better readability:

```
        # Add "validation-requested" if a tracker is new and all affects are new
        if (
            not self.tracker.external_system_id
            or "validation-requested" in self.tracker.meta_attr.get("labels", [])
        ) and (
            set(self.tracker.affects.all().values_list("affectedness", flat=True))
            == {Affect.AffectAffectedness.NEW}
        ):
            self._query["fields"]["labels"] += ["validation-requested"]
```

Reply:
Fixed, thank you.